scripts/tag_release.sh

```diff
@@ -34,7 +34,7 @@ echo "press [y] to push the tags"
 read -n 1 confirm
 
 if [ "${confirm}" == "y" ]; then
-    git push origin "${_branch}" --tags
```

Comment:
I thought this syntax meant "push all tags related to this branch" but googling for that I couldn't find any evidence that that is the case.

Reply:
@xlson I think it means to push all tags, plus other refspecs you provide as arguments (so the release branch in this case) :)